helmfile.d/stacks/tekton.yaml

```diff
@@ -8,79 +8,76 @@ templates:
     name: tekton-rbac
     namespace: kube-system
     labels:
-      app: tekton-rbac
-      stage: bootstrap
+      app: tekton
+      component: tekton-rbac
```

Comment:
```suggestion
```
We don't need component labels as all get labels from their name (`name=<release-name>` by default. And you can set the `app: tekton` label in the `tekton` template then all except this one will have it.